### ISSUE TITLE
Default dataloader params have shuffle=False

### ISSUE THREAD

**Issue by spsancti** [Deci-AI/super-gradients#416]
### Describe the bug
By default, it does not pass shuffle=True to the dataloader, so SequentialSampler gets instantiated. In training on Imagenet it makes NN see only examples of a single class which quickly throws it out of the minima.
Passing shuffle=True to the dataloader params solves the issue

### To Reproduce
Minimal example:
```import super_gradients
from super_gradients import Trainer
from super_gradients.training import MultiGPUMode
from super_gradients.training import models
from super_gradients.training.dataloaders import imagenet_train, imagenet_val
from super_gradients.training.metrics import Accuracy

super_gradients.init_trainer()

dataloader_params = {"batch_size": 196}  # buggy params
# dataloader_params = {"batch_size": 196, "shuffle": True}  # non-buggy params


train_params = {"max_epochs": 1,
                "initial_lr": 0.001,
                "optimizer": "SGD",
                "optimizer_params": {"weight_decay": 0.0001, "momentum": 0.9, "nesterov": True},
                "loss": "cross_entropy",
                "train_metrics_list": [Accuracy()],
                "valid_metrics_list": [Accuracy()],
                "loss_logging_items_names": ["Loss"],
                "metric_to_watch": "Accuracy",
                "greater_metric_to_watch_is_better": True
                }

train_dataloader = imagenet_train(dataloader_params=dataloader_params)
val_dataloader = imagenet_val(dataloader_params=dataloader_params)

model = models.get("resnet50", pretrained_weights="imagenet", num_classes=1000)

trainer = Trainer(experiment_name="reproduce_shuffle_bug",
                  multi_gpu=MultiGPUMode.OFF,
                  device='cuda')

trainer.train(model=model,
              training_params=train_params,
              train_loader=train_dataloader,
              valid_loader=val_dataloader)
```

### Expected behavior
Accuracy not dropping

### Environment:
 - OS Linux 5.4.0-94-generic x86_64
 - Super Gradients version 3.0.0

**Comment by ofrimasad** [comment]
fixed in PR https://github.com/Deci-AI/super-gradients/pull/581 
version 3.0.4
